replica/table.cc

```diff
@@ -680,7 +681,8 @@ class single_storage_group_manager final : public storage_group_manager {
         : _t(t)
     {
         storage_group_map r;
-        auto cg = make_lw_shared<compaction_group>(_t, size_t(0), dht::token_range::make_open_ended_both_sides());
+        auto full_token_range = dht::token_range::make(dht::first_token(), dht::last_token());
```

Comment:
This isn't completely accurate, the real limits are (first_vnode, last_vnode].

But it may be good enough. With 256 tokens first_vnode and last_vnode are likely to be within 0.5% of the edge. 

But please add a comment.

Reply:
Maybe I can easily grab actual first and last token from the replication map. I will see. Otherwise, I will just explain it as you suggest.